docs/computed-properties.md

```diff
@@ -72,7 +72,7 @@ Consider the following problematic scenario:
 
 To clear, or "bust", the stored cache, you can use PHP's `unset()` function.
 
-Below is an example of an action called `createPost()` that, by creating a new post in the application, makes the `posts()` computed stale—meaning the `posts` computed property needs to be re-computed to include the newly added post:
```

Comment:
I would revert this, as they're called computed properties, so makes more sense to me to leave "computed" and "property" together.